lib/AnimationsSettings.vala

```diff
@@ -22,4 +22,11 @@ namespace AnimationsSettings {
 
         return enable_animations;
     }
+
+    /**
+     * Utility that returns the given duration or 0 if animations are disabled.
+     */
+    public inline uint get_animation_duration (uint duration) {
+        return enable_animations ? duration : 0;
```

Comment:
We have to either use `get_enable_animations` here (because the value from gsettings hasn't been read yet) or create some `init` function that will initialize the gsettings read. In case we go with the second solution, we should then change all `AnimationsSettings.get_enable_animations ()` instances with `AnimationsSettings.enable_animations`.